gravitee-am-identityprovider/gravitee-am-identityprovider-jdbc/src/main/java/io/gravitee/am/identityprovider/jdbc/configuration/JdbcIdentityProviderConfiguration.java

```diff
@@ -103,6 +104,12 @@ public void setDatabase(String database) {
         this.database = database;
     }
 
+    public void setSchema(String schema) {
+        this.schema = schema;
+    }
+
+    public String getSchema() {return schema;}
```

Comment:
suggestion: as it is specific to psql, I would not create an entry for it. Instead I would use the "Options".